File: leetcode/1672/MaximumWealth.java
class MaximumWealth {
    public static void main(String[] args) {
        Solution sl = new Solution();
        int[][] accounts = { { 1, 2, 3 }, { 3, 2, 1 } };
        System.out.println(sl.maximumWealth(accounts));
    }
}

class Solution {
    public int maximumWealth(int[][] accounts) {
        int max = 0;
        int sum = 0;
        for (int i = 0; i < accounts.length; i++) {
            sum = 0;
            for (int j = 0; j < accounts[i].length; j++) {
                sum += accounts[i][j];
            }
            if (sum > max) {
                max = sum;
            }
        }
        return max;
    }
}

File: leetcode/1672/MaximumWealth.js
var maximumWealth = function(accounts) {
    return Math.max(...accounts.map((cust)=>cust.reduce((acc,cur)=>acc+cur)));
    // return Math.max(...accounts.map((x) => { return x.reduce((a,b) => { return a + b} ) } ))
    // return accounts.reduce((accum, curr) => Math.max(accum, curr.reduce((a, b) => a + b)) , 0)
};

console.log(maximumWealth([[1,2,3],[3,2,1],[4,5,6]]
    ));